lib/MusicBrainz/Server/Data/ArtistCredit.pm

```diff
@@ -195,14 +195,6 @@ sub find_for_artist {
     );
 }
 
-sub _clean {
```

Comment:
I guess you can remove the `-also` part on `use namespace::autoclean -also => [qw( _clean )];` too.